packages/react-native-web/src/exports/Image/__tests__/index-test.js

```diff
@@ -174,6 +171,38 @@ describe('components/Image', () => {
       expect(onLoadEndStub.mock.calls.length).toBe(2);
     });
 
+    test('is called on update if "headers" are different', () => {
```

Comment:
**thought:** Similar vein to before. Having tests seems good, but not entirely sure what our best practice is here when it comes to forks.

Reply:
I'd say let's keep them and see how maintainers react